omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/api/Deletable.java

```diff
@@ -28,4 +28,16 @@
 	 */
 	void delete(String uuid, String reason, RequestContext context) throws ResponseException;
 	
+	/**
+	 * Undeletes the specified resource, which in the OpenMRS context means either unvoiding or
+	 * unretiring it
+	 * 
+	 * @param uuid
+	 * @param context
+	 * @throws ResponseException
+	 * @return unvoided/unretired object
+	 * @since 2.2.0
+	 */
+	Object unDelete(String uuid, RequestContext context) throws ResponseException;
```

Comment:
should be capitalized like undelete (no capital D). E.g. for consistency with unvoid, unretire, etc.